vars/odsOrchestrationPipeline.groovy

```diff
@@ -108,7 +109,7 @@ private withPodTemplate(String odsImageTag, IPipelineSteps steps, Closure block)
                 resourceLimitMemory: '1Gi',
                 resourceRequestCpu: '200m',
                 resourceLimitCpu: '1',
-                alwaysPullImage: true,
+                alwaysPullImage: "${alwaysPullImage}",
```

Comment:
Not sure if this still works like this - I think we are supposed to pass a boolean here, not a string.

Reply:
that works -. I tested ... 

![image](https://user-images.githubusercontent.com/40628552/82445024-6d763980-9aa4-11ea-9665-00dff12061ea.png)